README.md

```diff
@@ -1,16 +1,57 @@
-# helping_hands
+# Helping Hands
 
-A new Flutter project.
+Is an application that is aimed to enable people to get help from people and to help people in their neighborhood.
 
-## Getting Started
+The project is open source and licensed under the MIT License:
+Check the [LICENSE](LICENSE) file for further information
 
-This project is a starting point for a Flutter application.
+- The app is soly aimed towards helping people
+- Because of the current corona crisis we strive to develop the app as quickly as possible!
 
-A few resources to get you started if this is your first Flutter project:
 
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+### Features of the app:
 
-For help getting started with Flutter, view our
-[online documentation](https://flutter.dev/docs), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+- A Feed where users can see nearby requests
+- A map where people can see how many requests are around them
+- A screen where users can post new requests
+- A messenger where users can chat with other users
+- A profile page where people can edit their data
+- Safe authentication & server communication
```

Comment:
Secure communication and authentication should be a given, not a promoted feature.